src/plugins/remove_button/plugin.ts

```diff
@@ -41,7 +42,9 @@ export default function (this: TomSelect, userOptions: RBOptions) {
 	const html =
 		'<a href="javascript:void(0)" class="' +
 		options.className +
-		'" tabindex="-1" title="' +
+		'" tabindex="' +
```

Comment:
Could you make the fallback value `0` so that it is tabbable by default?

Reply:
@veryspry Done: https://github.com/gravitywiz/tom-select/pull/8/commits/409aa29566c4a33ef05ce43733271961c4eee04a

Also, in that, we can close the GPAS PR: https://github.com/gravitywiz/gp-advanced-select/pull/36
Since it will pick the default 0 for tabbability. 

Does that sound good to you?

Reply:
Thanks @saifsultanc!

I think it would be worth bumping the version explicitly in the GPAS just to make sure this change is picked up.

Reply:
@veryspry Yup, bumping the update for sure.. But no need for that explicit update as on the GPAS PR https://github.com/gravitywiz/gp-advanced-select/pull/36

Reply:
@veryspry Yup. Bumping for sure. But **not** that explicit change to do (https://github.com/gravitywiz/gp-advanced-select/pull/36)

Reply:
@saifsultanc understood 🙌